orga/app/components/routes/authenticated/campaign/report.hbs

```diff
@@ -11,10 +11,14 @@
     </div>
 
     <div class="campaign-details-header__report">
-       {{#if this.participationsCount}}
-        <CampaignParticipantsDetails @campaign={{@campaign}} @participationsCount={{this.participationsCount}} @sharedParticipationsCount={{this.sharedParticipationsCount}} />
-      {{else}}
+      {{#if (eq @campaign.type "ASSESSMENT")}}
         <CampaignCreationDetails @creationDate={{this.creationDate}} @creatorName={{this.creatorName}} />
+      {{else}}
+        {{#if this.participationsCount}}
+          <CampaignParticipantsDetails @campaign={{@campaign}} @participationsCount={{this.participationsCount}} @sharedParticipationsCount={{this.sharedParticipationsCount}} />
+        {{else}}
+          <CampaignCreationDetails @creationDate={{this.creationDate}} @creatorName={{this.creatorName}} />
+        {{/if}}
       {{/if}}
```

Comment:
ce que tu as fait fonctionne nickel 👍 
mais ca me dérange un peu d'avoir 2 fois 
`<CampaignCreationDetails @creationDate={{this.creationDate}} @creatorName={{this.creatorName}} />`

**suggestion**
*dans report.hbs*
```
{{#if this.participationsCountForProfileCollectionCampaign}}
        <CampaignParticipantsDetails @campaign={{@campaign}} @participationsCount={{this.participationsCount}} @sharedParticipationsCount={{this.sharedParticipationsCount}} />
{{else}}
        <CampaignCreationDetails @creationDate={{this.creationDate}} @creatorName={{this.creatorName}} />
{{/if}}
```

*dans le report.js*
```
get participationsCountForProfileCollectionCampaign() {
    return this.args.campaign.type === 'PROFILES_COLLECTION' ? this.participationsCount : false;
  }
```

Après ce que je propose n'est qu'une suggestion à améliorer !
De plus, ce code est amener à bouger encore donc c'est pas grave s'il reste tel quel 🙂